app/views/api/v2/settings/main.json.rabl

```diff
@@ -3,3 +3,7 @@ object @setting
 extends "api/v2/settings/base"
 
 attributes :value, :description, :category, :settings_type, :default, :created_at, :updated_at
+
+node :category_name do
+  _(@setting.category.constantize.humanized_category || @setting.category.gsub(/Setting::/,''))
```

Comment:
1. The block has the setting passed in (`node :category_name do |s|`), use that instead of referring to `@setting` here. I thought this was correct in the previous version.
1. `@setting.class` should be more direct than constantising via `@setting.category.constantize`